### ISSUE TITLE
Question about evaluation metrics (HSI evaluation)

### ISSUE THREAD

**Issue by justinday123** [Panopticon-FM/panopticon#2]
Hi. I'm really thanks for your result. I'm student studying about satellite models.

I have some question about your evaluation metrics.

Is there some reasons that you didn't evaluate on HSI benchmarks?(Such as Indian Pines)

Thanks. Have a nice day.

**Comment by LeWaldm** [comment]
Hi @justinday123,

Thank you for your interest in Panopticon.
 
Our evaluation focussed on the GeoBench benchmark, adding datasets that contain images from satellites with different wavelengths than S2 (since it is abundant in GeoBench). We used the HS datasets Corine and Hyperview to simulate other multi-spectral sensors like SuperDove and MODIS by spectral convolution.
 
An advantage of Panopticon is the ability to process arbitrary combinations of optical channels. We evaluated the performance of different numbers of randomly selected channels of the HS Corine dataset in Figure 3. We observed the performance plateauing for ≥9 channels, which is why we did not evaluate with all channels.
 
If you have any more questions, don't hesitate to ask.
 
 Best, Leonard

**Comment by justinday123** [comment]
Thank you for your response.
So, may I confirm if your intention was to conduct evaluation on the Corine dataset from SpectralEarth?
When you mention evaluation on HSI, does that mean the model was trained on HSI datasets (from SpectralEarth) and then evaluated on the SpectralEarth Corine dataset?

Thank you again for your kind answer.

**Comment by ando-shah** [comment]
Hi @justinday123 ,

 These are great questions! We did indeed train on the SpectralEarth dataset's unlabeled split, and evaluated on the Corine labels from the same. You can find the details of the dataset from [DLR here](https://geoservice.dlr.de/web/datasets/enmap_spectralearth).

We didnt evaluate on other HS datasets (except Hyperview) as we were not specifically trying to beat HS models, but rather that the model is invariant to channels, i.e. can work with any set of channels or sensors.

Hope that helps!
ando

**Comment by justinday123** [comment]
Thank you sincerely. I wish you all the best in everything going forward.

**Comment by justinday123** [comment]
Hi. I have some additional questions!

Does panopticon loads dinov2 pretrained weights?(where is the path for using it?)
Also, when loading the pre-trained weights (.pt file), do you discard or skip the patch_embed.proj layer? 
Thank you!
